pkg/gui/custom_commands.go

```diff
@@ -50,10 +52,13 @@ func (gui *Gui) createCommandMenu(customCommands []config.CustomCommand, command
 			return option.customCommand.InternalFunction()
 		}
 
+		if option.invokeShell {
+			option.command = gui.OSCommand.NewCommandStringWithShell(option.command)
```

Comment:
rather than mutate option.command here I would have a separate `resolvedCommand` variable that we use. This means that if we want to reuse `options` from outside this callback down the line there's no chance of the callback's code interfering

Reply:
I see. Okay, makes sense.